modules/exploits/linux/http/kaltura_unserialize_rce.rb

```diff
@@ -0,0 +1,101 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  include Msf::Exploit::Remote::HttpClient
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => 'Kaltura Remote PHP Code Execution',
+      'Description'    => %q{
+        This module exploits an Object Injection vulnerability in Kaltura.
+        By exploiting this vulnerability, unauthenticated users can execute
+        arbitrary code under the context of the web server user.
+
+        Kaltura has a module named keditorservices that takes user input
+        and then use it as an unserialized function parameter. The constructed
```

Comment:
Is this a last one ? Should I commit this "users" and send PR ? or just sit and wait ur PR ?